draft-ietf-cose-merkle-tree-proofs.md

```diff
@@ -160,21 +160,20 @@ while switching from one verifiable data structure to another, so long as both s
 This document establishes a registry of verifiable data structure proof types tags,
 with the following initial contents:
 
-| Identifier  | Proof Type   | Reference
+| Identifier  | Proof Type   | Proof Value | Reference
 |---
-|0            | N/A          |
-|TBD_2        | inclusion    | {{sec-generic-inclusion-proof}}
-|TBD_3        | consistency  | {{sec-generic-consistency-proof}}
+|0            | N/A          | N/A | N/A
+|TBD_2        | inclusion    | bstr or list of bstr | {{sec-generic-inclusion-proof}}
```

Comment:
Wouldn't it be better/simpler for this to ALWAYS be a list and sometimes only have 1 element?

Reply:
I agree, this was just my first attempt at an update... and I wanted to hear from @henkbirkholz and other CBOR / COSE experts on how best to address this.